### ISSUE TITLE
Expand the Style Guide to improve consistency in docs and comms

### ISSUE THREAD

**Issue by shannonxtreme** [kubernetes/website#28691]
We should expand the [Contributor Style Guide](https://kubernetes.io/docs/contribute/style/style-guide/) so that contributors have a reference for common terms and phrases. 

Some examples of what we could add: 
* Making the distinction between "deprecating" an API and "removing" an API (#28690)
* Saying "manifest" vs "definition" or "spec" and other terms with multiple technically correct alternatives
* Pod vs pods (capitalization especially w.r.t the plural version of a word)
* When to use inline codeblocks vs just saying the resource name (eg: `VerticalPodAutoscaler` vs Vertical Pod Autoscaler) and whether to use the word "object" or "resource" after a code-blocked mention (eg: Create a `Service` object vs Create a `Service`)

A lot of the groundwork already exists in the current Style Guide, but it could use expansion and potential reorganization. 

We could, for example, organize into sections based on type of style issue (grammar/accessibility & inclusion/formatting/API). 

This may be a long term priority but I thought it might be a good discussion point for a future meeting as it has the potential to be a pain to maintain as it grows, but would be invaluable for new contributors (like me!). 

/cc @sftim

**Comment by natalisucks** [comment]
@shannonxtreme I'd love to hear from you if you think this issue is worth reviving. At the moment, many of the relevant issues you've mentioned have been closed due to inactivity, and this umbrella issue would need an overhaul to have it linked to relevant work in flight (or not). While I do think this work is important, we can absolutely reflect as such in a new issue

**Comment by natalisucks** [comment]
Coming back to this issue, and in the interest of cleaning up some of our older work, I am in favour of closing this given the amount of inactivity in the linked relevant work, and the overhaul required to make this umbrella issue suitable for our current docs scope.

Happy to hear feedback from others!

**Comment by shannonxtreme** [comment]
Yeah I think we should close this. If I have more time down the line I'll bring it up at a sig meeting 😁 I think that we covered a lot of this in the linked issues so I see it as a win still!

**Comment by natalisucks** [comment]
Agreed, thanks @shannonxtreme! 🎉 
/close
